src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/util/ParserUtil.java

```diff
@@ -26,6 +30,7 @@
 public class ParserUtil {
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
     public static final String MESSAGE_INVALID_KEYWORD = "Keywords for all provided prefixes should not be empty.";
+    public static final String MESSAGE_INVALID_DATE = "Date not in correct format dd-MM-yyyy";
```

Comment:
Just because the deadline message is also like that
```suggestion
    public static final String MESSAGE_INVALID_DATE = "Date should be a valid date in dd-mm-yyyy hhmm format";
```